src/pages/data-reporting/reports-api/v3.4/index.mdx

```diff
@@ -430,6 +430,63 @@ https://api.livechatinc.com/v3.4/reports/chats/ranking \
 </Code>
 </Section>
 
+<Section>
+<Text>
+
+### Engagement
+
+Shows the distribution of chats based on the engagement during the specified period.
+
+#### Specifics
+
+|                     |                                                              |
+| ------------------- | ------------------------------------------------------------ |
+| **Method URL**      | `https://api.livechatinc.com/v3.4/reports/chats/engagement` |
+| **HTTP method**     | POST, GET                                                    |
+| **Required scopes** | `reports_read`                                               |
+
+#### Request
+
+See [available parameters and filters](#available-parameters-and-filters).
+
+#### Response
+
+| Field                         | Notes                                                        |
+|-------------------------------|--------------------------------------------------------------|
+| `total`                       | The total number of engagements in the specified date range. |
```

Comment:
Hm, I have a problem with that. `engagement` is uncountable; it's an abstract noun. Therefore, we cannot really say `the number of engagements`. `engagement interactions` maybe?

Reply:
In fact, these interactions are simply chats :D It's just to emphasize that this report is about engagement. How about

>The total number of chats regardless of engagement in the specified date.

?

Reply:
It's better, but "regardless of engagement" confused me. I think we shouldn't mention 'engagement' at all in param definitions. All these 3 options (manual_invite, immediate_start, auto_invite) are some form of engagement, but when we had 'engagement' mentioned directly in the description, I thought these chats were only the ones started from targeted messages.